setup.py

```diff
@@ -4,10 +4,16 @@
     name="spidermon",
     version="1.23.0",
     url="https://github.com/scrapinghub/spidermon",
+    project_urls={
+        "Documentation": "https://spidermon.readthedocs.io/",
+        "Source": "https://github.com/scrapinghub/spidermon",
+        "Tracker": "https://github.com/scrapinghub/spidermon/issues",
+        "Release notes": "https://spidermon.readthedocs.io/en/latest/changelog.html",
+    },
     author="Zyte",
     author_email="opensource@zyte.com",
     description=("Spidermon is a framework to build monitors for Scrapy spiders."),
-    long_description=("Spidermon is a framework to build monitors for Scrapy spiders."),
+    long_description=open("README.rst", encoding="utf-8").read(),
```

Comment:
We are missing it in Scrapy, but you should also set `long_description_content_type="text/x-rst"`

Reply:
That's why I just filed #459

Reply:
I saw it in other projects but wasn't sure if it was necessary! I just added it to the PR.